addons/hr_expense/tests/test_expenses.py

```diff
@@ -975,3 +975,33 @@ def test_create_report_name(self):
             sheet_names,
             "The report name should be 'New Expense Report, paid by (employee|company)' as a fallback",
         )
+
+    def test_update_expense_price_on_product_standard_price(self):
+        """
+        Tests that updating the standard price of a product will update all the un-submitted
+        expenses using that product as a category.
+        """
+        expense_category = self.env['product.product'].create({
+            'name': 'Expense Category',
```

Comment:
category is a bad choice of word imo, a thing only expense uses.
Better to use product as it makes more sense (at least in the back-end)